install/configuring_inventory_file.adoc

```diff
@@ -1589,6 +1589,11 @@ openshift_master_named_certificates=[{"certfile": "/root/STAR.openshift.com.crt"
 openshift_master_overwrite_named_certificates=true
 ----
 
+[IMPORTANT]
+====
+The `cafile` certificate is imported to the `ca-bundle.crt` file on the masters during installation or during redeployment of certificates. The `ca-bundle.crt` file is mounted to every pod that runs in {product-title}. Several {product-title} components automatically trust the named certificates by default when they access the `masterPublicURL` endpoint. If you omit the `cafile` option from the certificates parameter, the functionality of Web Console and several other components is reduced.
```

Comment:
There's no `cafile` parameter in above openshift_master_named_certificates option. We better to add a more clear example like:
openshift_master_named_certificates=[{"certfile": "/path/on/host/to/crt-file", "keyfile": "/path/on/host/to/key-file", "names": ["public-master-host.com"], "cafile": "/path/on/host/to/ca-file"}]

Reply:
On it.